### PR TITLE
Da cytology expansion update 2:  Animal friends (Adds 4 new creatures)

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
rough design doc: 
https://hackmd.io/jmH7FiWNRsSg1YsgQ0TP9Q?view

This PR adds 4 new cyto growable mobs. These mobs are mainly support type mobs rather then the straight combat mobs that have been the focus of previous updates.

The mobs are:

Secretary pigeon
yeastyglob
evil crab(reworked!)
tiziran cobraworm

This PR also changes some things on the back end. 

ai speech subtrees get support for datum emotes
all meats are split off from the bloated food.dmi into meat.dmi
crabs are now basic mobs.
yeastglob4
cobraworm64
pigeon64_anim
pinch1

## Why It's Good For The Game

While cytology has been well recieved the main complaints have been lack of content and the lack of mobs that are useful to the departments. 

This PR helps remedy that.

## Changelog

:cl:
expansion: A new creature to cytology: secretary pigeon
expansion: A new creature to cytology: yeastyglob
expansion: A new creature to cytology:  tiziran cobraworm
expansion: A reworked creature to cytology: evil crab
expansion:  A new reagent: nightcrawler enzymes, it deals burn damage and increases plant maturation speed.
expansion: A new type of soil: worm castings.
expansion: Added a new type of meat: pigeon meat.
/:cl:

### CODE REVIEW

**Comment by Krysonism** [comment]
holy conflicts

**Comment by optimumtact** [comment]
you should remove the blue slime extract

**Comment by Krysonism** [comment]
> Remove the ~blue~ gold slime extract, Cytology should be the way to get mobs going forward.
> 
I agree, the problem is, what do we replace it with?

**Comment by optimumtact** [comment]
Nothing, I think it's perfectly acceptable to leave cytology as a source of actually interesting mobs to interact with.

I would also not be oppossed to getting rid of the sentience potion and adding a brain transplanter machine that only works on mobs that are actually designed to be interesting when someone is in them.

**Comment by Krysonism** [comment]
I agree sentience potions suck in some ways, the most obvious one being that they are themed as magic potions rather than having a sci-fi theming.

It well worth thinking about how we want sentience to be achieved in the future.

**Comment by Valorium-RB** [comment]
> Nothing, I think it's perfectly acceptable to leave cytology as a source of actually interesting mobs to interact with.
> 
> I would also not be oppossed to getting rid of the sentience potion and adding a brain transplanter machine that only works on mobs that are actually designed to be interesting when someone is in them.

Not gonna put words in their mouth, but what I thought they meant by "what do we replace it with?" was "what do we replace the current effect of gold slime extracts with, if we're scrapping their mob creation effect?". Which, personally, I think is a reasonable question given how gold slimes have formerly been an important part of xenobiology. Would your position be to get rid of gold slimes entirely, and restructure the xenobiology tree entirely, or something else?

**Comment by optimumtact** [comment]
> 

this is getting wildly offtopic so this will be my last comment on this matter here, but the reaction will be removed and not replaced.

**Comment by Krysonism** [comment]
worm_casings_anim

working on the soil / worm trap

File: code/_onclick/drag_drop.dm [hunk 1]
```diff
@@ -115,3 +115,7 @@
 		middragtime = 0
 		middragatom = null
 	..()
+
+/client/MouseEntered(object, location, control, params)
+	. = ..()
+	SEND_SIGNAL(src, COMSIG_CLIENT_MOUSEENTERED, object, location, control, params)
```
Comment (LemonInTheDark) [inline code/_onclick/drag_drop.dm:121]: This is not cheap. Do we need to do this?

**Comment by ExcessiveUseOfCobblestone** [comment]
Was this waiting on a review

**Comment by Shadowflame909** [comment]
I'd be willing to lose gold slimes to see these mobs live

Live!!!!

**Comment by LemonInTheDark** [comment]
shadow it's been half a year

**Pull request closed without merging**